services/jobs/src/main/java/com/linkedin/openhouse/jobs/api/validator/impl/OpenHouseJobsApiValidator.java

```diff
@@ -40,6 +39,12 @@ public void validateCreateJob(CreateJobRequestBody createJobRequestBody) {
               "clusterId : provided %s, %s",
               createJobRequestBody.getClusterId(), ALPHA_NUM_UNDERSCORE_ERROR_MSG_HYPHEN_ALLOW));
     }
+    if (!createJobRequestBody.getJobConf().getMemory().matches(SPARK_MEMORY_FORMAT)) {
```

Comment:
does this means, it is a mandatory config?

Reply:
No, updated to add checks